content/docs/user-guide/components/reference/terrain/terrain-macro-material.md

```diff
@@ -60,19 +78,45 @@ This structure is used when sending out information about the macro material set
 
 | Request Name | Description | Parameter | Return | Scriptable |
 |-|-|-|-|-|
 | `GetTerrainMacroMaterialData` | Returns the `MacroMaterialData` structure assigned to the **Terrain Macro Material** component. | None | [MacroMaterialData](#macromaterialdata) | Yes |
+| `GetTerrainMacroColorImageSize` | Returns the height/width/depth of the color texture in pixels. | None | RHI::Size | No |
+| `GetMacroColorImagePixelsPerMeter` | Returns the number of color texture pixels per meter in world space. | None | Vector2 | No |
 
 ## TerrainMacroMaterialNotificationBus
 
-The `TerrainMacroMaterialNotificationBus` is also an internal system bus that is only intended for communication between the terrain renderer and the **Terrain Macro Material** component. The `TerrainMacroMaterialNotificationBus` EBus notifies listeners through the following notification functions.
+The TerrainMacroMaterialNotificationBus is an internal EBus used by the terrain rendering and editing systems to monitor changes to Terrain Macro Materials. Other systems generally do not need to use this EBus since nothing outside the terrain system should need any information about individual Terrain Macro Materials. However, if a use case arises, the following notification functions on the TerrainMacroMaterialNotificationBus EBus interface can be used to monitor Terrain Macro Material changes.
 
 | Notification Name | Description | Parameter | Return | Scriptable |
 |-|-|-|-|-|
 | `OnTerrainMacroMaterialCreated` | Notifies listeners when a new macro material has been created. | None | EntityId; [MacroMaterialData](#macromaterialdata) | Yes |
 | `OnTerrainMacroMaterialChanged` | Notifies listeners when a macro material has been changed. | None | EntityId; [MacroMaterialData](#macromaterialdata) | Yes |
 | `OnTerrainMacroMaterialDestroyed` | Notifies listeners when a macro material is removed. | None | EntityId | Yes |
 | `OnTerrainMacroMaterialRegionChanged` | Notifies listeners when the bounding area of the macro material changes. | None | EntityId; Old Region: Aabb; New Region: Aabb | Yes |
+
+## TerrainMacroColorModificationBus
+
+The TerrainMacroColorModificationBus is an internal EBus used by the terrain editing systems to modify the Terrain Macro Material color texture. This is a lower-level API, most systems should use the higher-level [Paint Brush](/docs/user-guide/components/reference/paintbrush/paintbrush) painting API to modify the texture.
```

Comment:
Same here and for TerrainMacroColorModificationNotificationBus below. Should these have the markdown applied?

Reply:
```suggestion
The `TerrainMacroColorModificationBus` is an internal EBus used by the terrain editing systems to modify the Terrain Macro Material color texture. This is a lower-level API, most systems should use the higher-level [Paint Brush](/docs/user-guide/components/reference/paintbrush/paintbrush) painting API to modify the texture.
```